.github/workflows/development-environment.yml

```diff
@@ -85,5 +115,7 @@ jobs:
       - name: Run bootstrap code
         env:
           STRAP_DEBUG: 1
+          # This ensures that the bootstrap code will test against this branch
+          CI_CHECKOUT_BRANCH: ${{ github.head_ref || github.sha }}
```

Comment:
I believe `github.head_ref` is only available on PR, thus, falling back to `github.sha` and that hopefully works on `master`.

Reply:
It works:
<img width="562" alt="image" src="https://user-images.githubusercontent.com/44410/158440311-aaf44456-5b5a-4849-821a-d5c66c43dd12.png">